docs/database/use-entity-framework-db-contexts.md

```diff
@@ -0,0 +1,564 @@
+
+:::zone-end
+
+To retrieve the `ExampleDbContext` object from the DI container and use the database:
+
+```csharp
+public class ExampleService(ExampleDbContext context)
+{
+    // Use context...
+}
+```
+
+## Use EF to create a database context and then enrich it
+
+Alternatively, you can create a database context and add it to the DI container using the standard EF <xref:Microsoft.Extensions.DependencyInjection.EntityFrameworkServiceCollectionExtensions.AddDbContext*> method, as commonly used in non-.NET Aspire projects:
+
+:::zone pivot="sql-server-ef"
+
+```csharp
+builder.Services.AddDbContext<ExampleDbContext>(options =>
+    options.UseSqlServer(builder.Configuration.GetConnectionString("database")
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+```
+
+:::zone-end
+:::zone pivot="postgresql-ef"
+
+```csharp
+builder.Services.AddDbContext<ExampleDbContext>(options =>
+    options.UseNpgsql(builder.Configuration.GetConnectionString("database")
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+```
+
+:::zone-end
+:::zone pivot="oracle-ef"
+
+```csharp
+builder.Services.AddDbContext<ExampleDbContext>(options =>
+    options.UseOracle(builder.Configuration.GetConnectionString("database")
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+```
+
+:::zone-end
+:::zone pivot="sql-server-ef"
+
+```csharp
+builder.Services.AddDbContext<ExampleDbContext>(options =>
+    options.UseMySql(builder.Configuration.GetConnectionString("database")
+        ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+```
+
+:::zone-end
+
+You have more flexibility when you create the database context in this way, for example:
+
+- You can reuse existing configuration code for the database context without rewriting it for .NET Aspire.
+- You can choose not to use EF Core context pooling, which may perform better in some circumstances. For more information, see [Use EF context pooling in .NET Aspire](#use-ef-context-pooling-in-net-aspire)
+- You can use dynamic connection strings. For more information, see [Use EF with dynamic connection strings in .NET Aspire](#use-ef-with-dynamic-connection-strings-in-net-aspire)
+- You can use [EF Core interceptors](/ef/core/logging-events-diagnostics/interceptors) to modify database operations. For more information, see [Use EF interceptors in .NET Aspire](#use-ef-interceptors-in-net-aspire)
+
+By default, a database context created this way doesn't include .NET Aspire features, such as telemetry and health checks. To add those features, each .NET Aspire EF client integration includes a method named `Enrich\<DatabaseSystem\>DbContext`. These enrich context methods:
+
+- Apply an EF settings object, if you passed one.
+- Configure connection retry settings.
+- Configure instrumentation, such as tracing and health checks.
```

Comment:
- Apply the recommended defaults, unless you've disabled them through the Aspire EF settings:
  - Enable tracing.
  - Enable health checks.
  - Enable connection resiliency.

Reply:
Fixed.